docs/second-screen/linux-wayland.md

```diff
@@ -0,0 +1,59 @@
+**Required Package Installation**
+
+First you need to ensure that wayvnc is installed.
+This requires installation of both the package aml and neatvnc from the arch user repository or wherever you get your packages.
+After installing these packages wayvnc should install successfuly.
+
+**Create A Suitable Output**
+First up create a mode called `HEADLESS-1` The output must be named this in order to work successfuly under Sway
+
+Add the following code to the config file for sway at the very bottom. 
+
+This config file can be found at ~/.config/sway/config
+
+```
+output HEADLESS-1 {
+	pos RIGHTMOST,0
+	mode 1408x1872
+	bg COLOR solid_color
+}
+```
+This creates a new headless output name at position x,y if you have multiple monitors take the position of the rightmost one and add its width to get the rightmost position where I should position my new display.
+Replace RIGHTMOST with this calculated rightmost point.
+Replace COLOR with the 9 digit color code for a color.
+
+Outputs from sway can be found with the following command.
+```
+> swaymsg -t get_outputs
+Output eDP-1 'Unknown 0x0791 0x00000000' (focused)
+  Current mode: 1920x1080 @ 60.012001 Hz
+  Position: 0,0
+```
+In this example I would replace RIGHTMOST with 0 + 1920 because this screen starts at 0 and has a width of 1920 meaning that our RIGHTMOST point would be 1920,0 to place the screen directly alongside.
```

Comment:
```suggestion
In this example I would replace RIGHTMOST with 0 + 1920 because this screen starts at 0 and has a width of 1920 meaning that our RIGHTMOST point would be [1920, 0] to place the screen directly alongside.
```